.github/workflows/deployment.yaml

```diff
@@ -45,23 +51,32 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
           role-session-name: githubDeploymentWorkflow
-          aws-region: ${{ env.aws_region }}
+          aws-region: ${{ secrets.AWS_REGION }}
```

Comment:
Can I ask why we're making this a secret? You've given a default value in the documentation already...does obscuring this value provide us any benefit?

Reply:
Updated to use vars.